simulation_runner/tests/integration/test_bootstrap_integration.py

```
@@ -0,0 +1,218 @@
+    def test_missing_fred_home_shows_clear_error(self, monkeypatch, tmp_path):
+        """Test that missing FRED_HOME shows a clear error message."""
+        # Remove FRED_HOME from environment
+        monkeypatch.delenv("FRED_HOME", raising=False)
+
+        # Create empty .env without FRED_HOME
+        env_file = tmp_path / ".env"
+        env_file.write_text("DATABASE_URL=postgresql://user:pass@localhost:5432/db\n")
+
+        # Build CLI
+        subprocess.run(
+            ["pants", "package", "simulation_runner:simulation-runner-cli"],
```

Ruff (0.14.3)

Starting a process with a partial executable path

(S607)